README.md

```diff
@@ -0,0 +1,42 @@
+# Virtual-Pet
```

Comment:
There are some important bits of info missing from this. Some things that would make it much better:
- installation instructions (ie cloning the repo and what commands are available),
- what technologies are required to use it, what the application dependencies are (`JavaScript`, `Jest` and `Node` in this case)
- a description of what the app actually does and what its features are. What are the rules for how the pet is fed, exercised, how long it will live, what are the max and min values for things like hunger, fitness etc.
- how the app is tested,
- what you would do if you had more time
- your name